### ISSUE TITLE
CDS 0.1.6 not installing after update to NC 18.0.6

### ISSUE THREAD

**Issue by muranava** [nextcloud/documentserver_community#146]
`[settings] Error: Exception: terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
Killed
 at <<closure>>

 0. /var/www/nextcloud/apps/documentserver_community/lib/LocalAppData.php line 78
    OCA\DocumentServer\Document\FontManager->OCA\DocumentServer\Document\{closure}("*** sensitive parameters replaced ***")
 1. /var/www/nextcloud/apps/documentserver_community/lib/Document/FontManager.php line 76
    OCA\DocumentServer\LocalAppData->getReadLocalPath(OC\Files\SimpleFS\SimpleFolder {}, Closure {})
 2. /var/www/nextcloud/apps/documentserver_community/lib/Migration/RebuildFonts.php line 46
    OCA\DocumentServer\Document\FontManager->rebuildFonts()
 3. /var/www/nextcloud/lib/private/Repair.php line 104
    OCA\DocumentServer\Migration\RebuildFonts->run(OC\Repair {})
 4. /var/www/nextcloud/lib/private/legacy/app.php line 981
    OC\Repair->run()
 5. /var/www/nextcloud/lib/private/Installer.php line 160
    OC_App::executeRepairSteps("documentserver_community", {step: "OCA\\Doc ... "})
 6. /var/www/nextcloud/apps/settings/lib/Controller/AppSettingsController.php line 443
    OC\Installer->installApp("documentserver_community")
 7. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 170
    OCA\Settings\Controller\AppSettingsController->enableApps(["documentserver_community"], [])
 8. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 99
    OC\AppFramework\Http\Dispatcher->executeController(OCA\Settings\Con ... {}, "enableApps")
 9. /var/www/nextcloud/lib/private/AppFramework/App.php line 125
    OC\AppFramework\Http\Dispatcher->dispatch(OCA\Settings\Con ... {}, "enableApps")
10. /var/www/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php line 47
    OC\AppFramework\App::main("OCA\\Settings\\ ... r", "enableApps", OC\AppFramework\ ... {}, {_route: "settin ... "})
11. <<closure>>
    OC\AppFramework\Routing\RouteActionHandler->__invoke({_route: "settin ... "})
12. /var/www/nextcloud/lib/private/Route/Router.php line 299
    call_user_func(OC\AppFramework\ ... {}, {_route: "settin ... "})
13. /var/www/nextcloud/lib/base.php line 1008
    OC\Route\Router->match("/settings/apps/enable")
14. /var/www/nextcloud/index.php line 38
    OC::handleRequest()

POST /index.php/settings/apps/enable
from 92.169.126.80 by muranext at 2020-06-09T20:13:22+00:00`

**Comment by st7ticv0id** [comment]
Confirmed, exactly the same situation.

**Comment by Liakoni** [comment]
The same problem for me too.

**Comment by lightlike** [comment]
Possible duplicate of #145

**Comment by muranava** [comment]
> Possible duplicate of #145

i don't think so as i have apps/documentserver_community/3rdparty/onlyoffice/documentserver/server/tools/allfontsgen and get:
ldd allfontsgen 
	linux-vdso.so.1 (0x00007ffd6a7a6000)
	libgraphics.so => not found
	libkernel.so => not found
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f0ded885000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f0ded668000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f0ded2c9000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f0deda9c000)
